metricbeat/module/coredns/_meta/kibana/7/dashboard/Metricbeat-coredns-overview.json

```diff
@@ -0,0 +1,1395 @@
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "area",
+                "valueAxis": "ValueAxis-1"
+              },
+              {
+                "data": {
+                  "id": "3",
+                  "label": "Average coredns.stats.dns.request.size.bytes.bucket.2047"
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "area",
+                "valueAxis": "ValueAxis-1"
+              },
+              {
+                "data": {
+                  "id": "4",
+                  "label": "Average coredns.stats.dns.request.size.bytes.bucket.4095"
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "area",
+                "valueAxis": "ValueAxis-1"
+              },
+              {
+                "data": {
+                  "id": "5",
+                  "label": "Average coredns.stats.dns.request.size.bytes.bucket.8291"
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "area",
+                "valueAxis": "ValueAxis-1"
+              }
+            ],
+            "times": [],
+            "type": "line",
+            "valueAxes": [
+              {
+                "id": "ValueAxis-1",
+                "labels": {
+                  "filter": false,
+                  "rotate": 0,
+                  "show": true,
+                  "truncate": 100
+                },
+                "name": "LeftAxis-1",
+                "position": "left",
+                "scale": {
+                  "mode": "normal",
+                  "type": "linear"
+                },
+                "show": true,
+                "style": {},
+                "title": {
+                  "text": "Size Bytes"
+                },
+                "type": "value"
+              }
+            ]
+          },
+          "title": "Request Size Bytes [Metricbeat CoreDNS] ECS",
+          "type": "line"
+        }
+      },
+      "id": "45d03ed0-53f1-11e9-b466-9be470bbd327-ecs",
+      "type": "visualization",
+      "updated_at": "2019-04-25T22:47:01.782Z",
+      "version": "WzI3LDFd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "Requests by Zone [Metricbeat CoreDNS] ECS",
```

Comment:
you can use the `coredns.stats.dns.request.count` instead of the document count to show this metrics.
Also, this metric is informed by `family`, `proto, `server` and `zone`.